packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -346,6 +337,7 @@ export class ProposedTransformationExplorer {
             }
 
             diffModel.clearChanges()
+            diffModel.currentPatchIndex = 0
```

Comment:
Now that I see the `clearChanges`, probably makes more sense to actually put this there